### PR TITLE
Load pool list from wiki.ellaism.org static pages

### PR DESCRIPTION
**Pull request by stevemulligan**
This change adds a some JS that loads the pool list from the wiki static pages.

I'm not sure who all has write access to the wiki. If it's open to the public then this should not be merged :)

### CODE REVIEW

**Comment by ellaismer** [comment]
@stevemulligan Yeah the wiki is open to the public for editing. We might want to fix that or find another way to make sure we don't include unwanted data in website.

File: _includes/head.html [hunk 1]
```diff
@@ -16,6 +16,6 @@
 </script>
 
 <!-- For all browsers -->
-<link rel="stylesheet" href="{{ base_path }}/assets/css/main.css">
+<link rel="stylesheet" href="/assets/css/main.css">
```
Comment (ellaismer) [inline _includes/head.html:19]: Why this base_path is removed? This will prevent it from being hosted in subdir.

File: _includes/masthead.html [hunk 2]
```diff
@@ -8,12 +8,7 @@
         <ul class="visible-links">
           <li class="masthead__menu-item masthead__menu-item--lg"><a href="{{ base_path }}/">{{ site.title }}</a></li>
           {% for link in site.data.navigation.main %}
-            {% if link.url contains 'http' %}
-              {% assign domain = '' %}
-              {% else %}
-              {% assign domain = base_path %}
-            {% endif %}
-            <li class="masthead__menu-item"><a href="{{ domain }}{{ link.url }}">{{ link.title }}</a></li>
+            <li class="masthead__menu-item"><a href="{{ link.url }}">{{ link.title }}</a></li>
```
Comment (ellaismer) [inline _includes/masthead.html:11]: And this as well.

File: _includes/toc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <aside class="sidebar__right">
 <nav class="toc" markdown="1">
-<header><h4 class="nav__title"><i class="fa fa-{{ include.icon | default: 'file-text' }}"></i> {{ include.title | default: site.data.ui-text[site.locale].toc_label }}</h4></header>
+<header><h4 class="nav__title"><i class="fa fa-{{ include.icon | default: 'file-alt' }}"></i> {{ include.title | default: site.data.ui-text[site.locale].toc_label }}</h4></header>
```
Comment (ellaismer) [inline _includes/toc:3]: file-alt doesn't seem to be a valid icon.

File: _pages/home.md [hunk 4]
```diff
@@ -38,7 +38,7 @@ monthly-update:
   - excerpt: "Monthly Community Update - April 2018 [Read Now](/2018/05/05/monthly-update-apr-2018/){: .btn}"
 ---
 
-{% include feature_row id="intro" type="center" %}
+{% include social.html %}
```
Comment (ellaismer) [inline _pages/home.md:41]: Any reason why we stop using feature_row but use hard coded social? The former looks simper to me.

File: _pages/home.md [hunk 5]
```diff
@@ -7,9 +7,9 @@ header:
   ctas:
     - cta_label: "<i class='fa fa-building'></i> Learn More"
       cta_url: "/install/"
-    - cta_label: "<i class='fa fa-github'></i> Github"
+    - cta_label: "<i class='fab fa-github'></i> Github"
       cta_url: "https://github.com/ellaism"
-    - cta_label: "<i class='fa fa-gear'></i> Roadmap"
+    - cta_label: "<i class='fa fa-cogs'></i> Roadmap"
```
Comment (ellaismer) [inline _pages/home.md:12]: Looks like a multi-gear icon.

File: _pages/home.md [hunk 6]
```diff
@@ -7,9 +7,9 @@ header:
   ctas:
     - cta_label: "<i class='fa fa-building'></i> Learn More"
       cta_url: "/install/"
-    - cta_label: "<i class='fa fa-github'></i> Github"
+    - cta_label: "<i class='fab fa-github'></i> Github"
```
Comment (ellaismer) [inline _pages/home.md:10]: Is this a typo?

File: _sass/_ellaism.scss [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+	padding-right: 15px;
+	font-size: 1.2rem;
+}
+
+.social-links .fa-github {
+	color: #000;
+}
+
+.social-links .fa-reddit {
+	color: orangered;
```
Comment (ellaismer) [inline _sass/_ellaism.scss:21]: Is this a valid color string? If not, can we use `#XXXXXX` instead?

File: _sass/_syntax.scss [hunk 8]
```diff
@@ -21,7 +21,8 @@ figure.highlight {
     padding: 0.5em;
     background-color: $lighter-gray;
     content: "\f121";
-    font-family: "fontawesome" !important;
+		font-family: 'Font Awesome 5 Free';
```
Comment (ellaismer) [inline _sass/_syntax.scss:24]: Wrong indentation.

File: assets/js/_main.js [hunk 9]
```diff
@@ -79,4 +79,7 @@ $(document).ready(function(){
     midClick: true // allow opening popup on middle mouse click. Always set it to true if you don't provide alternative source.
   });
 
+  // WikiLoader init
+	$(".wiki-loader").WikiLoader();
```
Comment (ellaismer) [inline assets/js/_main.js:83]: Indentation doesn't seem right.

**Review by ellaismer — COMMENTED** [review]
Some comments. We need to fix them before merging it to master.

And it looks huge IMHO. Would be great if we can split fontawesome upgrade and wiki loader implementation to different PRs.

**Comment by stevemulligan** [comment]
Going to create separate PRs for font-awesome 5 and the rest of the changes.

**Pull request closed without merging**